api/provider/aws/common.go

```diff
@@ -86,3 +89,24 @@ func deleteEntityTags(tagStore tag.Store, entityType, entityID string) error {
 
 	return nil
 }
+
+func listClusterNames(ecsapi ecsiface.ECSAPI, instance string) ([]string, error) {
+	input := &ecs.ListClustersInput{}
+
+	output, err := ecsapi.ListClusters(input)
```

Comment:
This should use [ListClustersPages](https://github.com/aws/aws-sdk-go/blob/master/service/ecs/api.go#L1511):

```
clusterNames = []string{}
fn := func(output *ListClustersOutput, lastPage bool) bool {
    for _, arn := range output.ClusterArns { ... }

    return !lastPage
}

if err :=  e.AWS.ECS.ListClustersPages(&ecs.ListClustersInput{}, fn); err != nil {
    return nil, err
}

return clusterNames, nil
```